packages/camera/camera_web/lib/src/types/camera_options.dart

```diff
@@ -121,24 +128,29 @@ class VideoConstraints {
       json['deviceId'] = <String, Object>{'exact': deviceId!};
     }
 
+    json['bitrate'] = bitrate;
+
     return json;
   }
 
   @override
-  bool operator ==(Object other) {
-    if (identical(this, other)) {
-      return true;
-    }
-
-    return other is VideoConstraints &&
-        other.facingMode == facingMode &&
-        other.width == width &&
-        other.height == height &&
-        other.deviceId == deviceId;
-  }
+  bool operator ==(Object other) =>
+      identical(this, other) ||
+      other is VideoConstraints &&
+          runtimeType == other.runtimeType &&
+          bitrate == other.bitrate &&
+          facingMode == other.facingMode &&
+          width == other.width &&
+          height == other.height &&
+          deviceId == other.deviceId;
 
   @override
-  int get hashCode => Object.hash(facingMode, width, height, deviceId);
+  int get hashCode =>
+      bitrate.hashCode ^
+      facingMode.hashCode ^
+      width.hashCode ^
+      height.hashCode ^
+      deviceId.hashCode;
```

Comment:
Why change Object.hash by XOR?

Reply:
This is auto generated stuff. I will change it to Object.hash

Reply:
Changed.